db/orders.js

```diff
@@ -0,0 +1,90 @@
+const { client } = require('./index')
+
+const getOrderById = async (id)=>{
+    try {
+        const {rows: [order]} = await client.query(`
+            SELECT * FROM orders
+            WHERE id=$1
+        `,[id])
+
+        return order
+    } catch (error) {
+        console.error(error)
+    }
+}
+
```

Comment:
To test this one, you could call `getOrderById(1)` right below, so that you can confirm that it works.

Reply:
And create data in postico right before you do so.